designer/client/form-details.js

```diff
@@ -51,7 +52,7 @@ class FormDetails extends React.Component {
   }
 
   handleIsFeedbackFormRadio = (e) => {
-    const isFeedbackForm = e.target.value;
+    const isFeedbackForm = str2bool(e.target.value);
 
     if (isFeedbackForm) {
       this.setState({ feedbackForm: true, selectedFeedbackForm: undefined });
```

Comment:
I know this wasn't your code, so you don't have to action it but would be nice to refactor here. (Feel free to dismiss this review comment, I wont block merge over it!) 

we can remove the entire if else block and just do this 
`this.setState({ feedbackForm: isFeedbackForm, selectedFeedbackForm: !isFeedbackForm ? undefined : this.state.selectedFeedbackForm  });`